src/pages/contributors/index.js

```diff
@@ -10,8 +11,12 @@ function ContributorsPage({sponsors, contributors, panelists, host}) {
     <Page
       title="JavaScript Air Contributors"
     >
+
+      <HeaderBar />
+
       <div className="container contributors-page">
-        <header className="contributors-page__header">
+
+        {/*<header className="contributors-page__header">
```

Comment:
Rather than commenting this out, let's just delete it. [more info](https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f)